src/styles.css

```diff
@@ -7,10 +7,22 @@ pre {
 
 .branding img {
   height: 2rem;
+  vertical-align: middle;
+  width: 36px;
+  height: 36px;
+}
+
+.branding .logo-text {
+  color: var(--clr-header-title-color,#fafafa);
```

Comment:
could we use the `core` tokens in here?

Reply:
I could do even better .. "#fafafa" only :)

Reply:
I got the style from clarity.design - but that will change soon so it ok to simplify it for now